docs/content/release_notes/changelog_1.11.0.md

```diff
@@ -1,5 +1,6 @@
 # 1.11.0 (January 2023)
 
 IBM® block storage CSI driver 1.11.0 added new support and enhancements.
+- Enhanced dynamic host definition
```

Comment:
reminder for us to check what is the official feature name

Reply:
updated